php/Plugin.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Plugin Utility class.
+ *
+ * @package phpdocplugin
+ */
+
+namespace Pods\phpdocplugin;
+
+/**
+ * Plugin utility class.
+ */
+class Plugin {
+
+	/**
+	 * Absolute path to the main plugin file.
+	 *
+	 * @var string
+	 */
+	protected $file;
+
+	/**
+	 * Absolute path to the root directory of this plugin.
+	 *
+	 * @var string
+	 */
+	protected $dir;
+
+	/**
+	 * Setup the plugin.
+	 */
+	public function __construct() {
+		$this->file = PHPDOCPLUGIN_FILE;
+		$this->dir  = dirname( PHPDOCPLUGIN_FILE );
+	}
+
+	/**
+	 * Plugin init function.
+	 *
+	 * @see WP init action.
+	 */
+	public function init() {
+		// todo - init actions here.
+	}
+
+	/**
+	 * Is WP debug mode enabled.
+	 *
+	 * @return boolean
+	 */
+	public static function is_debug() {
+		return ( defined( 'WP_DEBUG' ) && WP_DEBUG );
+	}
+
+	/**
+	 * Is WP script debug mode enabled.
+	 *
+	 * @return boolean
+	 */
+	public static function is_script_debug() {
+		return ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG );
+	}
+
+	/**
+	 * Return the current version of the plugin.
+	 *
+	 * @return mixed
+	 */
+	public function version() {
+		return $this->meta( 'Version' );
```

Comment:
There's a static function get_plugin_version() that returns the constant, is this method necessary?

Reply:
yes, this is redundant. Thanks.